src/main/java/com/mcmoddev/mmdbot/core/Utils.java

```diff
@@ -74,14 +74,14 @@ public static void sleepTimer() {
     }
 
     /**
-     * Borrowed from Darkhax's BotBase, all credit for the below methods go to him.
-     * The Bot Base repo and code is now deleted if you need it for reference Proxy has a copy. Dark may also have one.
+     * Get the OffsetDateTime from an Instant value, relative to UTC+0 / GMT+0.
+     * Overtakes the last system which used LocalDateTime and caused issues with the database.
```

Comment:
Not really a database issue, in my situation it was because the time and date when printed out in various command had no indication of its timezone (which people usually assume to be either in their timezone or in UTC+0), even though the timezone is set to the running machine's timezone (which for MMDBot/Kira, is PDT).

Reply:
Interesting. I'll adjust.